examples/complete/main.tf

```diff
@@ -11,8 +11,26 @@ module "vpc" {
   context = module.this.context
 }
 
-module "fms" {
-  source = "../.."
+module "firewall_manager" {
+  source = "../../"
+
+  providers = {
+    aws.admin = aws.admin
+    aws       = aws
+  }
+
+  context = module.this.context
```

Comment:
please remove the extra empty lines and move `context = module.this.context` to the last line of the module invocation (per our standard practices)